infrastructure/kube/keep-test/btcdiff/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+resources:
+  - btc-difficulty-statefulset.yaml
+
+namespace: default
+
+commonLabels:
+  app: maintainer
+  type: btc-difficulty
+  network: goerli
+
+configMapGenerator:
+  - name: btc-difficulty-maintainer-info
+    files:
+      - keyfile/btc-difficulty-maintainer-keyfile
+  - name: btc-difficulty-maintainer-config
+    files:
+      - config/btc-difficulty-maintainer-config.toml
+
+secretGenerator:
+  - name: btc-difficulty-maintainer-eth-password
```

Comment:
```suggestion
  - name: btc-difficulty-maintainer-eth-accounts-password
```

Reply:
Done in 8830a95a70ffe1160809cd2e9459a5a314c50812.